distribution/lib/Standard/Base/0.0.0-dev/src/IO.enso

```diff
@@ -2,6 +2,9 @@ import project.Any.Any
 import project.Data.Text.Text
 import project.Nothing.Nothing
 
+# needed for Standard.Test.Test_Reporter
+polyglot java import java.io.PrintStream
```

Comment:
Here the [question](https://github.com/enso-org/enso/pull/9997#discussion_r1608274630) is slightly different: shouldn't there be `IO.println newline='\n'` - e.g. have a way to print without `\n` and use that method rather than calling directly to `System.out.print` in `Standard.Test.Test_Reporter`?

Reply:
Yes, that would likely be better. I just did not want to modify a core builtin for a simple change in the Test lib and from my perspective there is no harm in using a Java method instead. From the Native Image perspective, modifying the builtin seems better.

Reply:
- reported as #10028

Reply:
Import moved to `Extra_Imports.enso` file in aaa5e9ec9e